Submodels/Legacy/Bezier_Only_Parameterizations.py

```
@@ -0,0 +1,670 @@
+        Extract key parameters of the reference profile from the thickness and camber distributions.
+    - GetThicknessControlPoints(b_8, b_15)
+        Calculate the control points for the thickness distribution Bezier curves.
+    - GetCamberControlPoints(b_0, b_2, b_17)
+        Calculate the control points for the camber distribution Bezier curves.
+    - ConvertBezier2AirfoilCoordinates(thickness_x, thickness, camber_x, camber, theta)
+        Convert Bezier curves to airfoil coordinates.
+    - FindInitialParameterization(reference_file)
+        Find the initial parameterization for the airfoil.
+    """
+
+    def __init__(self, *args):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)